Storage/Storage/Model/Customer+CoreDataProperties.swift

```diff
@@ -0,0 +1,45 @@
+import Foundation
+import CoreData
+
+
+extension Customer {
+
+    @nonobjc public class func fetchRequest() -> NSFetchRequest<Customer> {
+        return NSFetchRequest<Customer>(entityName: "Customer")
+    }
+
+    @NSManaged public var siteID: Int64
+    @NSManaged public var userID: Int64
+    @NSManaged public var dateCreated: Date
+    @NSManaged public var dateModified: Date?
+    @NSManaged public var email: String
+    @NSManaged public var username: String?
+    @NSManaged public var firstName: String?
+    @NSManaged public var lastName: String?
+    @NSManaged public var avatarUrl: String?
+    @NSManaged public var role: String
+    @NSManaged public var isPaying: Bool
+    @NSManaged public var billingFirstName: String?
+    @NSManaged public var billingLastName: String?
+    @NSManaged public var billingCompany: String?
+    @NSManaged public var billingAddress2: String?
+    @NSManaged public var billingCity: String?
+    @NSManaged public var billingState: String?
+    @NSManaged public var billingPostcode: String?
+    @NSManaged public var billingCountry: String?
+    @NSManaged public var billingPhone: String?
+    @NSManaged public var billingEmail: String?
+    @NSManaged public var billingAddress1: String?
+    @NSManaged public var shippingFirstName: String?
+    @NSManaged public var shippingLastName: String?
+    @NSManaged public var shippingCompany: String?
+    @NSManaged public var shippingAddress1: String?
+    @NSManaged public var shippingAddress2: String?
+    @NSManaged public var shippingCity: String?
+    @NSManaged public var shippingState: String?
+    @NSManaged public var shippingPostcode: String?
+    @NSManaged public var shippingCountry: String?
+    @NSManaged public var shippingPhone: String?
+    @NSManaged public var shippingEmail: String?
+
+}
```

Comment:
I completely agree with you and it's very annoying. We can't use single inverse relationship for multiple forward relationships (this is understandable for most cases when it's possible to have 2 relationships instead of always 1 in our case):

![Screen Recording 2020-12-16 at 13 12 46](https://user-images.githubusercontent.com/3132438/102335380-dc673280-3fa0-11eb-8961-bdb906aee933.gif)

We can create `customerBilling` and `customerShipping` (and `orderBilling` + `orderShipping`) but that will look very strange as well:

<img width="787" alt="image" src="https://user-images.githubusercontent.com/3132438/102335936-98286200-3fa1-11eb-850d-ddfecddd74b9.png">

(but no warnings here)